cocotb/regression.py

```diff
@@ -168,6 +168,11 @@ def _discover_tests() -> Iterable[Test]:
 
         modules = [s.strip() for s in module_str.split(',') if s.strip()]
 
+        tests = None
+        if test_str:
+            tests = [s.strip() for s in test_str.split(',') if s.strip()]
+            not_found_tests = []
```

Comment:
```suggestion
```

Reply:
Changed in 0d656990c6bc950012502e9d25ff767c6ea59468.